components/Formular/FormularAddRecipe.js

```diff
@@ -82,24 +108,25 @@ export default function RecipeForm({ onSubmit, formName, defaultData }) {
             </select>
           </label>
         </StyledInput>
-
         <StyledInput>
           <label htmlFor="time">Time: </label>
           <input
             id="time"
             name="time"
             type="text"
             defaultValue={defaultData?.time}
+            aria-labelledby="time"
           />
         </StyledInput>
         <StyledInput>
           <label htmlFor="servings">Servings: </label>
           <input
-            defaultValue={defaultData?.servings}
             id="servings"
             name="servings"
             type="number"
             min="1"
+            defaultValue={defaultData?.servings}
+            aria-labelledby="servings"
```

Comment:
Not needed.

Reply:
the defaultValue is needed for editing our recipe, so we only remove the aria-labeleddby, correct ;-) ?